src/pages/store/index.tsx

```diff
@@ -43,7 +44,7 @@ const StoreHomePage = ({ user: { credits }, view, collections }: HomePageProps)
           <div className={styles.collections}>
             {collections.map(collection => (
               <ItemCard
-                image={collection.items[0]?.picture ?? NoImage.src}
+                image={getDefaultMerchItemPhoto(collection.items[0]) ?? NoImage.src}
```

Comment:
I think `getDefaultMerchItemPhoto` should handle defaulting to `NoImage` so if we want to change the image we only have to change it in one function